packages/web-runtime/src/pages/resolvePrivateLink.vue

```diff
@@ -105,6 +106,7 @@ export default defineComponent({
         const sharePathSegments = mountPoint ? [] : [unref(resource).name]
         let tmpResource = unref(resource)
         while (!mountPoint) {
+          resourceIsInsideShare = true
```

Comment:
Would be sufficient to set `resourceIsInsideShare = !mountPoint` once after line 104 instead of setting it to `true` in each iteration.

Reply:
While you're at it, what do you think about `resourceIsShareRoot` or `resourceIsNestedInShare` as var name?